### ISSUE TITLE
Warning: statement is not reachable

### ISSUE THREAD

**Issue by zoujiaqing** [Pure-D/serve-d#102]
VSCode's code-d plugin install error:

```bash
Installing into /home/zoujiaqing/.vscode-oss/extensions/webfreak.code-d-0.22.0/bin
Removing old version
Removed old version
> git clone --recursive https://github.com/Pure-D/serve-d.git serve-d
Cloning into 'serve-d'...
> git checkout v0.6.0
Note: switching to 'v0.6.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at a3c0469 bump to 0.6.0
> /opt/dmd/freebsd/bin64/dub upgrade
Upgrading project in /usr/home/zoujiaqing/.vscode-oss/extensions/webfreak.code-d-0.22.0/bin/serve-d
Fetching dub 1.18.0 (getting selected version)...
Fetching vibe-d 0.9.0 (getting selected version)...
Fetching dfmt 0.11.0 (getting selected version)...
Fetching workspace-d 3.5.0 (getting selected version)...
Fetching dsymbol 0.9.2 (getting selected version)...
Fetching dscanner 0.8.0 (getting selected version)...
Fetching libdparse 0.13.1 (getting selected version)...
Fetching libddoc 0.7.0 (getting selected version)...
> /opt/dmd/freebsd/bin64/dub build --compiler=dmd
Invalid source/import path: /home/zoujiaqing/.dub/packages/dfmt-0.11.0/dfmt/bin
Running pre-generate commands for dfmt...
Invalid source/import path: /home/zoujiaqing/.dub/packages/dscanner-0.8.0/dscanner/bin
Running pre-generate commands for dscanner...
Performing "$DFLAGS" build using dmd for x86_64.
diet-complete 0.0.3: building configuration "library"...
stdx-allocator 2.77.5: building configuration "library"...
emsi_containers 0.8.0-alpha.19: building configuration "library"...
eventsystem 1.2.0: building configuration "library"...
libdparse 0.13.1: building configuration "library"...
libddoc 0.7.0: building configuration "lib"...
dunit 1.0.16: building configuration "library"...
painlesstraits 0.3.0: building configuration "unittest"...
painlessjson 1.4.0: building configuration "library"...
cachetools 0.3.1: building configuration "library"...
requests 1.0.15: building configuration "std"...
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d(111,5): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d(115,5): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d(119,5): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-121(121,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-122(122,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-123(123,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-124(124,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-126(126,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-127(127,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-129(129,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-130(130,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-131(131,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-132(132,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-133(133,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-134(134,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-135(135,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-136(136,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-137(137,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-138(138,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-139(139,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-140(140,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-141(141,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-142(142,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-143(143,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-144(144,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-145(145,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-146(146,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-147(147,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-148(148,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-149(149,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-150(150,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d-mixin-151(151,1): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d(153,5): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d(154,5): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d(155,5): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d(156,5): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d(157,5): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d(158,5): Warning: statement is not reachable
/home/zoujiaqing/.dub/packages/requests-1.0.15/requests/source/requests/ssl_adapter.d(161,5): Warning: statement is not reachable
dmd failed with exit code 1.
Failed to install serve-d (Error code 2)
```

**Comment by WebFreak001** [comment]
can you send `/opt/dmd/freebsd/bin64/dub --version` and `dmd --version` output?

**Comment by zoujiaqing** [comment]
Thanks, dmd version is letest:
```bash

[zoujiaqing@freebsd ~/projects/hunt-skeleton]$ dmd --version
DMD64 D Compiler v2.093.0
Copyright (C) 1999-2020 by The D Language Foundation, All Rights Reserved written by Walter Bright
```

**Comment by WebFreak001** [comment]
can you check what the .so filenames are for the openssl (libssl and libcrypto) shared libraries? If you have multiple (like symlinks with different versions) send all the filenames.

**Comment by ikod** [comment]
Hello, @zoujiaqing 

should be fixed in requests 1.1.6, tested with
FreeBSD freebsd 12.1-RELEASE FreeBSD 12.1-RELEASE r354233 GENERIC  amd64
openssl-1.1.1g,1

**Comment by WebFreak001** [comment]
@zoujiaqing you will need to specify `"d.servedReleaseChannel": "nightly"` in your vscode user settings to make code-d fetch the current master commit instead of the v0.6.0 tag.
